src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
```

Comment:
A very general name, you can call it 'states'. If the function will be larger in size it may be a problem.